src/vision.cs

```diff
@@ -115,6 +116,14 @@ enum VNBarcodeSymbology {
 		Upce,
 	}
 
+	[TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+	[Native]
+	enum VNRequestRevision : ulong {
+		Unspecified = 0,
+		One = 1,
+		Two = 2,
+	}
```

Comment:
hmm... not sure I like this.

The main value of using an `enum` over `nuint` constants is that it conveys, thru intellisense, the correct value(s) for each API.

* If API `A` only has one revision, that `ARevision.One` is the only value and we won't be showing `Two` (no ambiguity);

* If API `B` has two revisions then both `BRevision.One` and `BRevision.Two` will be presented in the IDE and the later one can be annotated with availability attributes, e.g. `[IOS (12,0)]`. That makes it pretty clear.

The current scheme might be better at code sharing but IMO the resulting API are not easier to use. In fact the lack of availability (or constants) makes it even harder for developer to use the correct values.